doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

```diff
@@ -97,6 +97,14 @@ The following changes are mandatory to make your application work in the same wa
   * The PSA config is now validated by ncs/nrf/ext/oberon/psa/core/library/check_crypto_config.h.
     Users with invalid configurations must update their PSA configuration according to the error messages that check_crypto_config.h provides.
 
+* For :ref:`lib_wifi_credentials` library and Wi-Fi samples:
+
+  * :kconfig:option:`CONFIG_WIFI_CREDENTIALS_BACKEND_PSA_UID_OFFSET` has been removed because it was specific to the previous solution that used PSA Protected Storage instead of PSA Internal Protected Storage.
```

Comment:
```suggestion
  * :kconfig:option:`CONFIG_WIFI_CREDENTIALS_BACKEND_PSA_UID_OFFSET` has been removed because it was specific to the previous solution that used PSA protected storage instead of PSA Internal Trusted Storage (ITS).
```
Should _Internal Trusted Storage (ITS)_ be used in this case? If so please update this in the below sentences also.